src/app/helps/help.jsx

```diff
@@ -51,8 +51,6 @@ export default function HelpCard({ help }) {
   return (
     <Card
       sx={{
-        minWidth: 260,
-        minHeight: 230,
```

Comment:
can you keep th minWith and minHeight https://mosa3ada-git-fork-styrexx-fix-cardhelpstack-biztech.vercel.app/

Reply:
@soufyanelf I can let the minHeight but  I removed minWidth because is having a problem with the phone screen